x-pack/solutions/search/packages/shared-ui/src/form_info_field/form_info_field.tsx

```diff
@@ -33,61 +35,82 @@ export const FormInfoField: React.FC<FormInfoFieldProps> = ({
   copyValue,
   dataTestSubj,
   copyValueDataTestSubj,
+  maxWidth,
+  minWidth,
 }) => {
   const { euiTheme } = useEuiTheme();
 
   return (
-    <EuiFlexGroup gutterSize="s" alignItems="center" responsive={false}>
+    <EuiFlexGroup
+      gutterSize="s"
+      alignItems="center"
+      wrap
+      style={{
+        maxWidth: maxWidth ? `${maxWidth}px` : undefined,
+        minWidth: minWidth ? `${minWidth}px` : undefined,
+      }}
+    >
       {label && (
         <EuiFlexItem grow={false}>
           <EuiTitle size="xxxs">
             <h1>{label}</h1>
```

Comment:
I know this isn't your change, but we shouldn't be using an <h1> here as it's not a main page title.